nparse.py

```diff
@@ -19,8 +19,8 @@
 os.environ['QT_SCALE_FACTOR'] = str(
     config.data['general']['qt_scale_factor'] / 100)
 
-
-CURRENT_VERSION = '0.6.4'
+# todo - set this to appropriate value
+CURRENT_VERSION = '0.6.4-DLV'
```

Comment:
Yeah i'd just leave this out for now, we will do a version bump commit once a release is ready

Reply:
ok.  You're the version keeper.

Reply:
reverted to 0.6.4 in latest push